gdcdictionary/schemas/virus_genome_variation.yaml

```diff
@@ -4,7 +4,7 @@ id: "virus_genome_variation"
 title: Virus Genome Variation
 type: object
 namespace: https://chicagoland.pandemicresponsecommons.org
-category: data_file
+category: genomic_data_file
```

Comment:
i see in the dictionary in QA that "Virus SRA Run Archive" is of category `data_file`, not `genomic_data_file`

Reply:
The `Virus SRA Run Archive` should be `genomic_data_file`. I will push another commit.